diff -ru ../LHEInterface.orig/plugins/LHEWriter.cc ./plugins/LHEWriter.cc
--- ../LHEInterface.orig/plugins/LHEWriter.cc	2010-02-25 01:30:53.000000000 +0100
+++ ./plugins/LHEWriter.cc	2013-11-06 13:50:57.872886875 +0100
@@ -29,9 +29,11 @@
 
     private:
 	std::ofstream	file;
+        std::string filename;
 };
 
-LHEWriter::LHEWriter(const edm::ParameterSet &params)
+LHEWriter::LHEWriter(const edm::ParameterSet &params):
+filename(params.getParameter<std::string>("ouputFile"))
 {
 }
 
@@ -44,7 +46,7 @@
 	edm::Handle<LHERunInfoProduct> product;
 	run.getByLabel("source", product);
 
-	file.open("writer.lhe", std::fstream::out | std::fstream::trunc);
+	file.open(filename.c_str(), std::fstream::out | std::fstream::trunc);
 	std::copy(product->begin(), product->end(),
 	          std::ostream_iterator<std::string>(file));
 }
